R/tm_variable_browser.R

```diff
@@ -1,37 +1,36 @@
-#' Variable Browser Teal Module
+#' Variable browser `teal` module
 #'
-#' The variable browser provides a table with variable names and labels and a
-#' plot that visualizes the content of a particular variable.
-#' specifically designed for use with `data.frames`.
+#' Module provides provides a detailed summary and visualization of variable distributions
+#' for `data.frame` objects, with interactive features to customize analysis.
 #'
-#' @details Numeric columns with fewer than 30 distinct values can be treated as either factors
-#' or numbers with a checkbox allowing users to switch how they are treated (if < 6 unique values
-#' then the default is categorical, otherwise it is numeric).
+#' Numeric columns with fewer than 30 distinct values can be treated as either discrete
+#' or continuous with a checkbox allowing users to switch how they are treated(if < 6 unique values
+#' then the default is discrete, otherwise it is continuous).
 #'
 #' @inheritParams teal::module
 #' @inheritParams shared_params
-#' @param parent_dataname (`character(1)`) If this `dataname` exists in `datasets_selected`
-#'   then an extra checkbox will be shown to allow users to not show variables in other datasets
-#'   which exist in this `dataname`.
-#'   This is typically used to remove `ADSL` columns in `CDISC` data. In non `CDISC` data this
-#'   can be ignored. Defaults to `"ADSL"`.
-#' @param datasets_selected (`character`) A vector of datasets which should be
-#'   shown and in what order. Names in the vector have to correspond with datasets names.
-#'   If vector of length zero (default) then all datasets are shown.
-#'   Note: Only datasets of the `data.frame` class are compatible; using other types will cause an error.
+#' @param parent_dataname (`character(1)`) string specifying a parent dataset.
+#' If it exists in `datasets_selected`then an extra checkbox will be shown to
+#' allow users to not show variables in other datasets which exist in this `dataname`.
+#' This is typically used to remove `ADSL` columns in `CDISC` data.
+#' In non `CDISC` data this can be ignored. Defaults to `"ADSL"`.
+#' @param datasets_selected (`character`) vector of datasets which should be
+#' shown, in order. Names must correspond with datasets names.
+#' If vector of length zero (default) then all datasets are shown.
+#' Note: Only `data.frame` objects are compatible; using other types will cause an error.
 #'
 #' @aliases
-#'   tm_variable_browser_ui,
-#'   tm_variable_browser_srv,
-#'   tm_variable_browser,
-#'   variable_browser_ui,
-#'   variable_browser_srv,
-#'   variable_browser
+#' tm_variable_browser_ui,
+#' tm_variable_browser_srv,
+#' tm_variable_browser,
+#' variable_browser_ui,
+#' variable_browser_srv,
+#' variable_browser
```

Comment:
Why would someone search for `tm_variable_browser_ui`. I'm not a fan of aliases, because: 
- Why would I search for something which doesn't exist?
- Why would I see duplicated documentation entries in the package index?

Reply:
I agree,I was unsure if there was any specific reason for it, as it wasn't defined for any other modules. Nonetheless, I will remove it.

Reply:
Just to clarify why I'm so against it

<img width="549" alt="Skärmavbild 2024-02-26 kl  10 00 50" src="https://github.com/insightsengineering/teal.modules.general/assets/6959016/6f5221fa-87ab-4355-8fd3-a10faabfafca">

Reply:
yes, it could end up causing more confusion rather than any help.